booking_tools.py

```
@@ -0,0 +1,147 @@
+
+BOOKING_STATUS = Literal["cfrm", "pndng"]
+
+from shotgrid_rest import ShotGridRest
+
+
+async def _handle_errors(coro):
+    try:
+        return await coro
+    except httpx.HTTPStatusError as e:
+        return {"error": f"ShotGrid API Error: {e.response.status_code}", "message": e.response.text}
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)